assets/pages/new-contributions-landing/components/ContributionFormContainer.jsx

```diff
@@ -98,6 +99,7 @@ function ContributionFormContainer(props: PropTypes) {
     : (
       <div className="gu-content__content">
         <h1 className={headerClasses}>{countryGroupDetails.headerCopy}</h1>
+        { props.countryGroupId === 'UnitedStates' ? <ContributionUsTicker /> : null }
```

Comment:
minor: could we make `UnitedStates` a const in this repo... somewhere...? Just to avoid magic strings.

Reply:
yes good point. these are already in `countryGroup.js`, and there's an export `countryGroup` with appropriate types. (https://github.com/guardian/support-frontend/blob/master/assets/helpers/internationalisation/countryGroup.js#L33-L86).

in this case you would use it as follows:

```javascript
countryGroup[props.countryGroupId].name === 'United States'
```

which is a bit roundabout, so maybe it's worth replicating the object keys as a property `id` so you could do

```
props.countryGroupId === countryGroup.UnitedStates.id
```